### PR TITLE
140 recommendations 

### PR DESCRIPTION
**Pull request by olierxleben**
Recommendations now working

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -56,6 +56,21 @@ export const App: FC<{}> = () => {
     setCover(value);
   };
 
+  const handleShare = async (shareData: SharingOptions) => {
+    const canShare = (await Share.canShare()).value;
+    console.log(shareData);
```
Comment (MarcoRehmer) [inline src/App.tsx:61]: console.log can be removed

File: src/models/index.ts [hunk 2]
```diff
@@ -16,16 +16,21 @@ export interface Issue {
...
   articleId: string;
   position: number;
   title: string;
+  headline: string;
+  provider: string;
+  description: string;
   teaser: string;
   source: string;
   url: string;
+  link: string;
```
Comment (MarcoRehmer) [inline src/models/index.ts:30]: whats the different between "link" and "url"?

File: src/pages/ArticleTeaser.tsx [hunk 3]
```diff
@@ -1,153 +1,200 @@
...
   }
 
   function getShareIcon(): string {
-    return shareOutline
+    return shareOutline;
+  }
+
+  async function handleShare() {
+    const shareData = await getStaticContent("share-article");
+    console.log(shareData);
```
Comment (MarcoRehmer) [inline src/pages/ArticleTeaser.tsx:106]: console.log can be removed

File: src/models/index.ts [hunk 4]
```diff
@@ -16,16 +16,21 @@ export interface Issue {
...
   imageUrl: string;
+  image_credit: string;
```
Comment (MarcoRehmer) [inline src/models/index.ts:32]: why snake-case not camel-case? :-)

File: src/App.tsx (hunk 1 above)
Comment (olierxleben) [reply src/App.tsx:61]:
```suggestion
```

File: src/pages/ArticleTeaser.tsx (hunk 3 above)
Comment (olierxleben) [reply src/pages/ArticleTeaser.tsx:106]:
```suggestion
```

File: src/models/index.ts (hunk 4 above)
Comment (olierxleben) [reply src/models/index.ts:32]: I used image_credit in the beginning on firebase.

File: src/models/index.ts (hunk 2 above)
Comment (olierxleben) [reply src/models/index.ts:30]: I kind of merged the interfaces. Within the Pages there was another interface "Article". It is an intermediate step here. But thanks for remembering: I added another issue #164 to finish refactoring.

**Review by MarcoRehmer — APPROVED** [review]

**Pull request merged**